test/StockportContentApiTests/Unit/Repositories/LandingPageRepositoryTests.cs

```diff
@@ -1,15 +1,22 @@
-﻿namespace StockportContentApiTests.Unit.Repositories;
+﻿using StockportContentApi.Model;
 
-public class LandingPageRepositoryTest
+namespace StockportContentApiTests.Unit.Repositories;
+
+public class LandingPageRepositoryTests
 {
     private readonly LandingPageRepository _repository;
-    private readonly EventRepository _eventRepository;
+    private readonly Mock<EventRepository> _eventRepository;
     private readonly Mock<IContentfulClient> _contentfulClient = new();
     private readonly Mock<IContentfulFactory<ContentfulLandingPage, LandingPage>> _contentfulFactory = new();
     private readonly Mock<IContentfulFactory<ContentfulNews, News>> _newsFactory = new();
     private readonly Mock<IContentfulFactory<ContentfulProfile, Profile>> _profileFactory = new();
+    private readonly Mock<IContentfulFactory<ContentfulEvent, Event>> _eventFactory;
+    private readonly Mock<IContentfulFactory<ContentfulEventHomepage, EventHomepage>> _eventHomepageFactory;
+    private readonly Mock<ITimeProvider> _timeprovider;
+    private readonly Mock<ICache> _cacheWrapper;
```

Comment:
for variables that just get initialised in the controller, pass them straight where you need to use them or create new on in line with the ones above these